app/uk/gov/hmrc/exports/repositories/DeclarationRepository.scala

```diff
@@ -81,4 +84,10 @@ class DeclarationRepository @Inject()(mc: ReactiveMongoComponent, appConfig: App
       .remove("id" -> declaration.id, "eori" -> declaration.eori)
       .map(_ => Unit)
 
+  def deleteExpiredDraft(expiryDate: Instant): Future[WriteResult] = {
```

Comment:
Please can we not expose `WriteResult` to the service layer as it is a database layer concern

It should have a count field on it which we can return, otherwise I'd suggest returning `Future[Unit]`

Reply:
Good point.  It's just the count field I'm interested in (for logging)

Reply:
...alternatively, the repo class could log the outcome (i.e. how many records deleted)

Reply:
I'd just expose the count with `Future[Int]`

Reply:
That's what I went with